src/components/Staking/StakingStatsBox.tsx

```diff
@@ -1,78 +1,78 @@
 // Import libraries
-import React, { useState, useEffect } from "react"
-import styled from "@emotion/styled"
+import React, { useState, useEffect, ReactNode } from "react"
 import { useIntl } from "react-intl"
-import { Spinner } from "@chakra-ui/react"
+import { Code, Flex, Icon, Spinner } from "@chakra-ui/react"
 // Import components
 import Translation from "../Translation"
 import Tooltip from "../Tooltip"
 import Link from "../Link"
-import Icon from "../Icon"
 // Import utilities
 import { Lang } from "../../utils/languages"
 import { getData } from "../../utils/cache"
 import { getLocaleForNumberFormat } from "../../utils/translations"
+import { MdInfoOutline } from "react-icons/md"
 
 // Constants
 const NA_ERROR = "n/a"
 const ZERO = "0"
 const MAX_EFFECTIVE_BALANCE = 32
 
-// Styled components
-const Container = styled.div`
-  display: flex;
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    flex-direction: column;
-  }
-`
-
-const Cell = styled.div`
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  padding: 1rem 2rem;
-  border-left: 1px solid ${({ theme }) => theme.colors.preBorder};
-  @media (max-width: ${({ theme }) => theme.breakpoints.m}) {
-    border-left: none;
-    border-top: 1px solid #33333355;
-  }
-  &:first-child {
-    border-left: none;
-    border-top: none;
-  }
-`
-
-const Value = styled.code`
-  font-weight: 700;
-  font-size: 2rem;
-  background: none;
-  display: flex;
-  align-items: center;
-  text-align: center;
-  text-transform: uppercase;
-  color: ${({ theme }) => theme.colors.primary};
-`
+const Cell: React.FC<{ children: ReactNode }> = ({ children }) => {
+  return (
+    <Flex
+      direction="column"
+      alignItems="center"
+      py={4}
+      px={8}
+      borderLeft={{ base: "none", md: "1px solid" }}
+      borderLeftColor={{ md: "preBorder" }}
+      borderTop={{ base: "1px solid #33333355", md: "none" }}
+      sx={{
+        "&:first-child": {
+          borderLeft: "none",
+          borderTop: "none",
+        },
+      }}
```

Comment:
I believe all of this can be removed in favor of Chakra's [Divider component](https://chakra-ui.com/docs/components/divider/usage), given the color `preBorder` (as the hex value used for `borderTop` can barely be seen in dark mode), and removing all of this as the component renders the `hr` element with ability to change orientation.
```suggestion
```
For repetition, the `Divider` could be rendered in a child component called `CellDivider` say.

```tsx
const CellDivider = () => {
  // Hook needed because the `orientation` prop does not directly accept responsive values
  const orientation = useBreakpointValue<"horizontal" | "vertical">({
    base: "horizontal",
    md: "vertical",
  })
  return <Divider borderColor="preBorder" orientation={orientation} />
}
```